apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/QueryUtils.java

```diff
@@ -178,4 +221,59 @@ public static boolean isInRange(Instant lastUpdated, DateRangeParam range) {
     }
     return true;
   }
+
+  /**
+   * Convert an integer value into claims data available BitSet.
+   *
+   * @param maskVal integer value suitable for decomposing into a Claims BitSet.
+   * @return {@link BitSet} denoting which claims have data.
+   */
+  public static BitSet convertClaimsBtimaskValue(int maskVal) {
```

Comment:
`convertClaimsBtimaskValue` -> `convertClaimsBitmaskValue`

Reply:
fix; thx for catching that.